DashAI/front/src/api/datasets.ts

```diff
@@ -16,18 +16,13 @@ export const getDatasets = async (): Promise<IDataset[]> => {
 export const updateDataset = async (
   id: number,
   name: string | undefined = undefined,
-  taskName: string | undefined = undefined,
```

Comment:
Were you able to compile the front end and see if it would run with this modification?

If the dataset name was used in any component, then the front end is likely to crash.
And if it doesn't crash, isn't there some subcomponent left somewhere that doesn't show anything?